pkg/operator/ceph/nfs/config.go

```diff
@@ -46,8 +62,35 @@ func getRadosURL(n *cephv1.CephNFS, nodeID string) string {
 	return url
 }
 
+func (r *ReconcileCephNFS) generateKeyring(n *cephv1.CephNFS, name string) error {
+	osd_caps := fmt.Sprintf("allow rw pool=%s", n.Spec.RADOS.Pool)
```

Comment:
```suggestion
	osdCaps := fmt.Sprintf("allow rw pool=%s", n.Spec.RADOS.Pool)
```

Golang uses camelcase.

Reply:
Fixed.